### ISSUE TITLE
403 response, result of consent screen

### ISSUE THREAD

**Issue by Orawko** [ytdl-js/react-native-ytdl#77]
I would like to use some youtube video in my app, but when I run:

```
const youtubeURL = 'http://www.youtube.com/watch?v=04GiqLjRO3A';
const urls = await ytdl(youtubeURL, { quality: 'highestaudio' });
console.log(urls)
```
I recieve 403 network error, and following warning:

`Possible Unhandled Promise Rejection (id: 0):
TypeError: Cannot read property 'storyboards' of undefined
TypeError: Cannot read property 'storyboards' of undefined
    at Object.exports.getStoryboards`

Week or two earlier, I used that without any problems. Maybe something has changed on the youtube site.

Edit:
When I paste that url to safari on simulator, I can see youtube consent screen. As I have found, It is shown to some European countries (in my case Poland) 

<img src="https://user-images.githubusercontent.com/36421825/119798599-944b2500-bedb-11eb-9d86-94744de00e48.png" width="300">

**Comment by Darguima** [comment]
This problem has been resolved in the latest version `v4.8.2`

Just need to reinstall the package 

> yarn upgrade react-native-ytdl@latest

**Comment by Orawko** [comment]
@Darguima Thank You for the response! After updating to mentioned version, problem is gone and video loads succesfully (maybe slightly longer than earlier)
